Samples/DynamicLighting/Core/DebugObject.cs

```diff
@@ -1,60 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-
-using OpenTK;
-using OpenTK.Graphics.OpenGL;
-
-using Duality;
-using Duality.Components;
-using Duality.Resources;
-using Duality.ColorFormat;
-using Duality.VertexFormat;
-
-namespace Debug
-{
-	[Serializable]
-	[RequiredComponent(typeof(Transform))]
-    public class DebugObject : Renderer
```

Comment:
I believe this file was not used anymore but was still in source control. Since net sdk projects include all .cs files by default this revealed the leftover .cs file.